docs/encyclopedia/workers.mdx

```diff
@@ -190,6 +191,30 @@ Once an Activity Task finishes execution, the Worker responds to the Temporal Se
 - ActivityTaskTerminated
 - ActivityTaskTimedOut
 
+## What is a Nexus Operation Task? {#nexus-operation-task}
+
+A Nexus Operation Task provides the context needed to proceed with an Nexus Operation Task Execution.
+Nexus Operation Tasks largely represent the Nexus Operation Scheduled Event, containing the data needed to execute a Nexus Operation handler.
+
+## What is a Nexus Operation Task Execution? {#nexus-operation-task-execution}
+
+An Nexus Operation Task Execution occurs when a Worker uses the context provided from the Nexus Operation Task and executes the Nexus Operation handler (also known as the Nexus Operation handler function).
+
+The NexusOperationScheduled Event corresponds to when the Temporal Service puts the Nexus Operation Task into the Task Queue.
+
+The NexusOperationStarted Event corresponds to when the Worker picks up the Nexus Operation Task from the Task Queue.
+
+Either NexusOperationCompleted or one of the other Closed Nexus Operation Events corresponds to when the Worker has yielded back to the Temporal Service.
+
+The API to schedule a Nexus Operation Execution provides an "effectively once" experience, even though there may be several Nexus Operation Task Executions that take place to successfully complete an Nexus Operation.
+
+Once an Nexus Operation Task finishes execution, the Worker responds to the Temporal Service with a specific Event:
+
+- NexusOperationCompleted
+- NexusOperationFailed
```

Comment:
```suggestion
- NexusTaskFailed
```

Reply:
Should we document [this level](https://github.com/temporalio/api/blob/d761393206a8d6d41878fe436a8690ee7a21783d/temporal/api/workflowservice/v1/request_response.proto#L1593) of the Nexus internals, since they are one level below what user's typically interact with?

Reply:
This was also correct, there's no NexusTaskFailed event.